test_scripts/Polices/Validation_of_PolicyTables/077_ATF_usage_and_error_counts_update_minutes_in_hmi_limited.lua

```diff
@@ -25,18 +25,22 @@
 -- Expected result:
 -- SDL must: increment value of "minutes_in_hmi_limited" for this <X+N> minutes in Local Policy Table.
 ---------------------------------------------------------------------------------------------
-
---[[ General configuration parameters ]]
-Test = require('connecttest')
-local config = require('config')
 config.defaultProtocolVersion = 2
 config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.isMediaApplication = true
```

Comment:
@istoimenova, In activateAppInSpecificLevel() function in line 291 used "SDL.ActivateApp" request with parameter "level". Please notice such parameter is not available for "SDL" interface, but available for "BC"